cmd/janitor.go

```diff
@@ -52,6 +52,8 @@ Janitor can be used in several ways.
 		RunE: cli.NewHandler().Janitor.RunE,
 		Args: cli.NewHandler().Janitor.Args,
 	}
+	cmd.Flags().Int(cli.Limit, 10000, "Limits the number of records retrieved from database for deletion.")
```

Comment:
So this limit here is 10K instead of 100k as suggested in the comment?

Reply:
I think you can adjust the defaults to what has worked for you

Reply:
Actually, my limit was 1 million records, but it takes an hour for the select only and delete is a long process (not affecting database resources nor service though). I tought that a lower default would be more reasonable for common use cases.

Reply:
Then we stick with the lower limits :)

Reply:
Well then, we just have to decide between 10k and 100k.
In my database setup a single 100 batch delete for consent (the slowest table, I guess because of many cascade deletes) takes 1 second or so. A default limit of 10k would take a total of 100 seconds, while a limit of 100k would take 1000 seconds (i.e. almost 17 minutes). We could stick with one of these (I would take the 100k limit, if I have to choose) or get a value in the middle.

Reply:
@aeneasr what is you opinion on this?